src/components/header.css

```diff
@@ -0,0 +1,63 @@
+.header {
+  width: 100%;
+  height: 100px;
+  position: fixed;
+}
+
+.headerContainer {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 10rem;
+  height: 100px;
+  align-items: center;
+}
+
+.navList {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.linkLi {
+  list-style: none;
+  margin: 0;
+  margin-left: 1rem;
+  letter-spacing: 1.25px;
+  text-transform: uppercase;
+  font-size: 16px;
```

Comment:
It looks like they are using 15px on the current site. I actually like 16px a bit better, but let's go ahead and update it to `15px` to match.